### ISSUE TITLE
Better link foreground color for focused problem

### ISSUE THREAD

**Issue by octref** [microsoft/vscode#89933]
Ref: #11847

Link's foreground color does not good contrast against focused background:

![image](https://user-images.githubusercontent.com/4033249/73669432-45310f00-465c-11ea-806c-939295aafb13.png)

![image](https://user-images.githubusercontent.com/4033249/73669435-46623c00-465c-11ea-9732-171b00b334ba.png)

![image](https://user-images.githubusercontent.com/4033249/75404624-fc1d5500-58bf-11ea-9da1-c5cd0ecec35b.png)

**Comment by octref** [comment]
I tested dark/light/hc/nord.

![image](https://user-images.githubusercontent.com/4033249/75404442-6d103d00-58bf-11ea-817a-6b6ce76212e0.png)

![image](https://user-images.githubusercontent.com/4033249/75404450-71d4f100-58bf-11ea-8606-9779b98d01ee.png)

![image](https://user-images.githubusercontent.com/4033249/75404463-78fbff00-58bf-11ea-9f54-e78ae8020a3e.png)

![image](https://user-images.githubusercontent.com/4033249/75404474-80230d00-58bf-11ea-9402-58e193474361.png)

**Comment by RMacfarlane** [comment]
The color contrast ratio when hovering on the link in Light+ is 1.3:1, I think we're aiming for 4.5:1. Dark actually also fails, it's 4.0:1 right now.

**Comment by octref** [comment]
I have decided to not have any hover effect on focused links for the upcoming release.

![image](https://user-images.githubusercontent.com/4033249/75584017-acf03500-5a23-11ea-9022-a003e71d60dd.png)

I think this is fine since the underline still indicates the link and people can hover over the link to see tooltip.

@joaomoreno ``textLink.foreground` seems to have bad contrast to tree's `list.activeSelectionBackground`. Is there a place where we already render link in active selection in a tree? Or should I introduce a new color?
